src/main/java/cms/gov/madie/measure/config/AddEndorsementsChangeUnit.java

```diff
@@ -11,7 +11,7 @@
 import java.util.Arrays;
 import java.util.List;
 
-@ChangeUnit(id = "add_updated_endorsements_initializer", order = "1", author = "madie_dev")
+@ChangeUnit(id = "add_updated_endorsements_initializer_2", order = "1", author = "madie_dev")
```

Comment:
would change ID re-trigger this? if yes any side effects to prod?

Reply:
We want it to re-trigger to reset that table with the new values.